app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -267,6 +276,13 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
         }
     }
 
+    private fun setLocate(Lang: String) {
```

Comment:
should it not be setLocale

Reply:
SIr, here I have used a custom function setLocate. setLocale object has been created inside this function

Reply:
use a more meaningful name instead of setLocate

Reply:
@batbrain7 please check it now